src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -16,82 +16,71 @@ type Props = {
   tooltipToggle?: any;
   focussed: boolean;
   iconRight?: any;
+  checked?: boolean;
+  value?: any;
+  changeChecked?: any;
+  onCheckedChange: any;
 };
 
 const styleVariables = loadStyleVariables();
 
-const Toggle = styled.div`
-  position: relative;
-  display: inline-block;
+const ToggleInput = styled.input<{ checked?: boolean; value?: any, changeChecked?: void }>`
+  z-index: 200;
+  opacity: 0;
   width: 40px;
-  height: 20px;
-  span {
-    position: absolute;
-    cursor: pointer;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    border: 2px solid ${styleVariables.gray};
-    color: ${styleVariables.colorWhite};
-    background: ${styleVariables.grayLight};
-    border-radius: 30px;
-    transition: all 0.4s;
-    :after {
-      top: 2px;
-      left: 5px;
-      font-size: 10px;
-      content: "O";
-      visibility: visible;
-      display: block;
-      position: absolute;
-      transform: translateX(20px);
-    }
+  height: 15px;
+  position: absolute;
+  &:checked ~ label span {
+    transform: translatex(30px);
+    transition: transform 0.2s ease-in;
   }
-  input:checked + span {
+  &:checked ~ label{
     background: ${styleVariables.green};
     border: 2px solid ${styleVariables.greenDark};
-    :after {
-      content: "I";
-      visibility: visible;
-      display: block;
-      position: absolute;
-      transform: translateX(2px);
-    }
+    transition: transform 0.2s ease-in;
+  }
+
+  &:checked ~ label p{
+    opacity: 1;
+    content: I;
+    transform: translateX(-20px);
+    color: ${styleVariables.colorWhite};
   }
 `;
 
-const Input = styled.input`
-  position: absolute;
-  cursor: pointer;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  z-index: 90;
+const Label = styled.label<{ checked?: boolean; value?: any, changeChecked?: void}>`
+  display: inline-block;
+  width: 40px;
+  height: 15px;
   border-radius: 30px;
-  transition: all 0.4s;
-  &:before {
-    background-color: ${styleVariables.colorWhite};
-    position: absolute;
-    border: 2px solid ${styleVariables.gray};
-    content: "";
-    width: 25px;
-    height: 25px;
-    left: -6px;
-    top: -7px;
-    border-radius: 24px;
-    transition: all 0.4s;
-  }
-  &:checked {
-    transform: translateX(20px);
-  }
-  &:checked:before {
-    border: 2px solid ${styleVariables.greenDark};
-  }
+  position: relative;
+  cursor: pointer;
+  background: ${styleVariables.grayLight};
+  border: 2px solid ${ styleVariables.gray };
+
+`;
+
+const Switch = styled.span<{ checked?: boolean; value?: any, changeChecked?: void}>`
```

Comment:
Er wordt niets gedaan met checked en changeChecked in deze styled component? Mag weg dan :)